vector-config/src/main/res/values-v29/config-settings.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- This file contains values to show or hide some settings, and default values for some settings
+     - boolean keys ending with "_visible" set the visibility of the setting
+     - boolean keys ending with "_default" set the default value of the setting
+     When a setting is hidden, the default value still applies
+     -->
+
+    <!-- Level 1: Labs -->
+    <bool name="settings_labs_enable_voice_broadcast_visible">true</bool>
```

Comment:
That's a nice solution, hence could be hard to understand for fork maintainers.
I would rather test API version in the code, to eventually hide the setting, even if the `bool` is set to `true`.